lib/services.cf

```diff
@@ -239,6 +239,10 @@ bundle agent standard_services(service,state)
       "service_enabled" -> { "CFE-2923" }
         expression => returnszero( "$(call_systemctl) is-enabled $(service) > /dev/null 2>&1", useshell);
       "service_active"  expression => reglist(@(systemd_service_info), "ActiveState=active");
+      "service_active"  or => {
```

Comment:
Shouldn't the promise on line 241 be redacted since this performs the same check?

Reply:
Or, can we consolidate them into a single call?
```
      "service_active"  expression => reglist(@(systemd_service_info), "ActiveState=(active|activating)");
```